src/lib.rs

```diff
@@ -47,10 +47,9 @@ extern crate once_cell;
 pub use self::docbuilder::RustwideBuilder;
 pub use self::docbuilder::DocBuilder;
 pub use self::docbuilder::options::DocBuilderOptions;
-pub use self::docbuilder::metadata::Metadata;
 pub use self::web::Server;
 
-pub mod error;
+pub(crate) mod error;
```

Comment:
Since this module is defined in `lib.rs`, we don't need `pub(crate)`. All other modules are descendants of the top-level module and can access its private modules.

```suggestion
mod error;
```